include/xlsxwriter/chartsheet.h

```diff
@@ -524,7 +524,7 @@ lxw_error chartsheet_set_footer_opt(lxw_chartsheet *chartsheet,
                                     const char *string,
                                     lxw_header_footer_options *options);
 
-lxw_chartsheet *lxw_chartsheet_new();
+lxw_chartsheet *lxw_chartsheet_new(void);
```

Comment:
I think this should not be `void` but `lxw_worksheet_init_data *init_data`. See here:
https://github.com/jmcnamara/libxlsxwriter/blob/8d50d03b65db6382436156426474d9f16fe90471/src/chartsheet.c#L28